docs/source/user/RELEASE.rst

```diff
@@ -14,6 +14,43 @@ Features
 ========
 Wheel, egg, and tar.gz files are available on `pypi <https://pypi.org/project/sasdata/>`_.
 
+New in Version 0.9.0
+--------------------
+This is an enhancement release with updates to the unit conversion routines, the ability to load data from URIs, the
+addition of a wedge slicer and other slicer enhancements.
+
+What's Changed
+^^^^^^^^^^^^^^
+
+Feature Enhancements
+____________________
+* Refactor nxsunit by @krzywon in https://github.com/SasView/sasdata/pull/13
+* Enable the sector slicing to allow both sides independantly by @butlerpd in https://github.com/SasView/sasdata/pull/36
+* 31: Load data from URIs by @krzywon in https://github.com/SasView/sasdata/pull/37
+* SasData counterpart to SasView SlicerExtension_1344 by @jack-rooks in https://github.com/SasView/sasdata/pull/61
```

Comment:
Not sure what 31: means (maybe the issue #?). As it stands it is confusing. I recommend removing this (and all other instances below (e.g. line 40 and 41 etc).